src/library_exceptions.js

```diff
@@ -121,7 +121,7 @@ var LibraryExceptions = {
     this.get_exception_ptr = function() {
       // Work around a fastcomp bug, this code is still included for some reason in a build without
       // exceptions support.
-      var isPointer = {{{ exportedAsmFunc('___cxa_is_pointer_type') }}}(this.get_type());
```

Comment:
Is this change related?

Reply:
Its related, since now we simply don't need `exportedAsmFunc`.     However, I wonder if I can split this part out..

Reply:
Spit out into #18588

Reply:
Rebased.  This is no longer part of this change.